nrpy/infrastructures/CarpetX/simple_loop.py

```diff
@@ -0,0 +1,104 @@
+"""
+Simple loop generation for use within the CarpetX infrastructure.
+
+Author: Zachariah B. Etienne
+Email: zachetie **at** gmail **dot* com
+Contributor: Ken Sible
+Email: ksible *at* outlook *dot* com
+"""
+
+from typing import List
+
+
+def simple_loop(
+    loop_body: str,
+    enable_simd: bool = False,
+    loop_region: str = "",
+    loop_centering: List[int] = [0, 0, 0],
+    run_on_device: bool = True,
```

Comment:
Missing docstring. Also what is meant by device? Be clear!

Reply:
Fixed docstrings